### PR TITLE
Remove exclude external lb label from control-plane nodes

### PR DESCRIPTION
**Pull request by pperiyasamy**
The latest [metallb](https://github.com/metallb/metallb/pull/2073) skips nodes having the label `"node.kubernetes.io/exclude-from-external-load-balancers": ""` (particularly set on the control plane nodes) from advertising bgp routes for external service. But this breaks ovnk service related tests. Hence this PR removes the label from those nodes after metallb deployment on kind cluster, so that these nodes can also act as bgp speaker for external services.

Fixes https://github.com/ovn-org/ovn-kubernetes/issues/4104

### CODE REVIEW

File: contrib/kind.sh [hunk 1]
```diff
@@ -1050,6 +1050,16 @@ install_metallb() {
...
   delete_metallb_dir
 
+  # The metallb commit https://github.com/metallb/metallb/commit/1a8e52c393d40efd17f28491616f6f9f7790a522
+  # removes control plane node from acting as a bgp speaker for service routes.
+  # Hence remove node.kubernetes.io/exclude-from-external-load-balancers label from control-plane nodes
+  # so that they are also available for advertising bgp routes which are needed for ovnkube's service
+  # specific e2e tests.
+  MASTER_NODES=$(kind get nodes --name "${KIND_CLUSTER_NAME}" | sort | head -n "${KIND_NUM_MASTER}")
+  for n in $MASTER_NODES; do
+    kubectl label node "$n" node.kubernetes.io/exclude-from-external-load-balancers-
```
Comment (tssurya) [inline contrib/kind.sh:1060]: nice! thanks for the quick fix.

**Review by tssurya — APPROVED** [review]
LGTM. Once CI passes will merge this. This is certainly a side effect of consuming recent metalLB version perhaps.

**Comment by tssurya** [comment]
re-running the LGW job again to ensure we are good.

**Review by trozet — APPROVED** [review]

**Pull request merged**